spec/support/helpers.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require File.expand_path("../path.rb", __FILE__)
```

Comment:
is this require necessary? `spec_helper.rb` requires everything in `spec/support` already